cron/poll_pull_requests.py

```diff
@@ -79,16 +79,17 @@ def poll_pull_requests(api):
             # the PR is mitigated or the threshold is not reached ?
             if variance >= threshold or not is_approved:
                 voting_window = gh.voting.get_extended_voting_window(api, settings.URN)
-                if vote_total >= threshold / 2:
+                if settings.IN_PRODUCTION and vote_total >= threshold / 2:
                     # check if we need to mention the meritocracy
                     try:
                         commit = pr["head"]["sha"]
                         if not db.query("SELECT * FROM meritocracy_mentioned WHERE commit_hash=?",
                                         (commit,)):
+                            meritocracy_mentions = meritocracy - {pr["user"]["login"].lower()}
```

Comment:
GitHub automatically fixes mention cases. Try mentioning me like this: `@plasmapower`. You'll notice it turns into `@PlasmaPower`.